pleasew

```diff
@@ -69,4 +69,4 @@ for x in `ls "$DIR"`; do
 done
 ln -sf "${DIR}/please" "${LOCATION}/plz"
 echo -e >&2 "${GREEN}Should be good to go now, running plz...${RESET}"
-exec "$TARGET" "$@"
+exec "$TARGET" "$PLZ_ARGS" "$@"
```

Comment:
This is an error if it's undefined? (we set +u at the top of the script)

I think something like `"${PLZ_ARGS+}"` would help, but we also need to be sure it doesn't inject a spurious empty string into the arguments. Maybe just `${PLZ_ARGS+}` would do it?

Reply:
Silly me, I haven't actually tried it **without** `$PLZ_ARGS` 😄

Reply:
I think `${PLZ_ARGS:-}` will be the right syntax here.